packages/site/src/layout/NavMenu.module.css

```diff
@@ -0,0 +1,97 @@
+.navMenuContainer {
+    --hoverOffset: var(--space-smallest);
+    --negativeHoverOffset: calc(-1 * var(--hoverOffset));
+    --subItemInset: calc(var(--space-small) + var(--space-smaller));
```

Comment:
I can see a case for just doing these inline, but setting them as vars (I hope) makes it more explicit what they're doing

Side note for Future Chris: the calc to get 12px is one of the most common "custom CSS" patterns I've seen, even in our own components, so look into supporting that value as a default token